style.css

```diff
@@ -0,0 +1,23 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+.main-section {
+  border-top: 8px solid #F5C85F;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  padding: 40px;
+  flex-wrap: wrap;
+}
+
+.vertical-line {
+  display: flex;
+  margin: 10px;
+  height: 200px;
+  width: 3px;
+  background: linear-gradient(transparent, #FFD700, transparent);
+  margin: 30px;
```

Comment:
I would recommend you add the margin only to the sides and not in the top or bottom like this: `margin: 0 30px;`